src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -63,7 +63,7 @@ internal partial class CurlHandler : HttpMessageHandler
         private bool _preAuthenticate = false;
         private CredentialCache _credentialCache = null;
         private CookieContainer _cookieContainer = null;
-        private bool _useCookie = false;
+        private bool _useCookie = true;
```

Comment:
This should be pulled out into that shared defaults file.  I'm ok if that's done in a follow-up PR.